Source/JavaScriptCore/dfg/DFGOperations.cpp

```diff
@@ -2921,14 +2933,19 @@ JSC_DEFINE_JIT_OPERATION(operationStringReplaceStringGeneric, JSString*, (JSGlob
     JITOperationPrologueCallFrameTracer tracer(vm, callFrame);
     auto scope = DECLARE_THROW_SCOPE(vm);
 
+    JSValue replaceValue = JSValue::decode(encodedReplaceValue);
+    if (replaceValue.isString()) {
+        if (JSString* result = tryReplaceOneCharUsingString<>(globalObject, stringCell, searchCell, asString(replaceValue)))
+            OPERATION_RETURN(scope, result);
+        OPERATION_RETURN_IF_EXCEPTION(scope, nullptr);
```

Comment:
Ditto.

Reply:
`OPERATION_RETURN` will release the `scope`.